assets/components/patronsEvents/patronsEvents.jsx

```diff
@@ -7,10 +7,19 @@ import React from 'react';
 import PageSection from 'components/pageSection/pageSection';
 import OtherProduct from 'components/otherProduct/otherProduct';
 
+import { getMemLink } from '../../pages/bundles-landing/helpers/externalLinks';
```

Comment:
This means that you are pointing from a shared component a page specific helper. I would change the location of `externalLinks` helper to be a global helper. Additionally, should `externalLink` be part of the routes files? It feels weird to have two files to handle paths/urls.

Reply:
Dammit I forgot to move that, it's supposed to be in the shared helpers folder, good catch!

Reply:
hahah it means that I am consistent through time. I wonder if it would be better to create one entity to handle paths and URLs.